plugins/samples/enable_recaptcha/tests.textpb

```diff
@@ -0,0 +1,8 @@
+test {
+  name: "Enable reCAPTCHA challenge"
+  num_chunks:10
```

Comment:
Can you explain a bit how the tests work? This tells the test runner to break the input body up into exactly 10 chunks? In a prod environment this is not deterministic right?

Reply:
You're right. This breaks the body into 10 chunks. In prod this behavior is not deterministic. This just tests that the plugin can handle potentially ugly chunking boundaries.